src/library/VUMeter.js

```diff
@@ -27,13 +28,13 @@ class VUMeter {
    *   queue, which controls the speed of VU meter decibel updates.
```

Comment:
Perhaps:

"The size of the FIFO queue which determines how fast the VUMeter gets updated"